app/client/ui/DocTour.ts

```diff
@@ -44,7 +48,17 @@ async function makeDocTour(docData: DocData, docComm: DocComm): Promise<IOnBoard
       return String(tableData.getValue(rowId, colId) || "");
     }
     const title = getValue("Title");
-    let body: HTMLElement | string = getValue("Body");
+
+    const bodyHtmlContent = sanitizeTutorialHTML(marked.parse(getValue("Body"), {
+      async: false, renderer
+    }));
+    const element = document.createElement('div');
+    element.innerHTML = bodyHtmlContent;
+
+
+    let body: HTMLElement = element;
+
+
```

Comment:
I think you can replace the `dom('p', body),` below to something like `dom('p', markdown(body)),` or `dom('p', el => el.innerHTML = bodyHtmlContent)`, depending on whether you're using the `markdown` helper or your own renderer.